axie-scholar-utilities/source/axie/breeding.py

```diff
@@ -138,6 +138,7 @@ def calculate_breeding_cost(self):
 
     def calculate_fee_cost(self):
         number_of_breeds = len(self.breeding_file)
+        cost = 0
```

Comment:
Redudant?

Reply:
if redundant I propose this use elif and else, will be in new commit

Reply:
fair enough